06-html5-apis/00_Storage_DnD_WebSockets/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>HTML APIs</title>	
+	<link rel="stylesheet" type="text/css" href="css/index.css">
+	<script type="text/javascript" src="js/index.js"></script>
+</head>
+
+<body>
+	<div class="container">
+		<section>
+		<fieldset>
+			<h1> Insert a new Player here! </h1>
+			<label for="name"> Player Name:</label><br>
+			<input id="playerName" type="text" name="name" placeholder="Insert player name"></input>
+			<br><br>
+			<label for="name"> Player Description:</label><br>
+			<textarea id="playerDescription" placeholder="Inser player description here"></textarea>
+			<br><br>
+			<button onclick="saveDataLocalStorage()">Save LocalStorage</button>
```

Comment:
I'd rather see the events attached in the js file than in the html, except if you are a big angular enthusiast ;)